### PR TITLE
role-update command

### PR DESCRIPTION
**Pull request by ArxdSilva**
Related to #1379

### CODE REVIEW

File: api/permission.go [hunk 1]
```diff
@@ -638,3 +638,40 @@ func listDefaultRoles(w http.ResponseWriter, r *http.Request, t auth.Token) erro
 	w.Header().Set("Content-Type", "application/json")
 	return json.NewEncoder(w).Encode(roles)
 }
+
+// title: list default roles
```
Comment (andrestc) [inline api/permission.go]: should be something like "update role"

File: auth/user.go [hunk 2]
```diff
@@ -277,6 +277,48 @@ func (u *User) AddRole(roleName string, contextValue string) error {
...
+	if ctx != "" {
+		role.ContextType, err = permission.ParseContext(ctx)
+		if err != nil {
+			return err
+		}
+	}
+	if desc != "" {
+		role.Description = desc
+	}
+	if newName == "" {
```
Comment (andrestc) [inline auth/user.go]: should this be `newName == "" || newName == name`?

**Comment by ggarnier** [comment]
Maybe. If something goes wrong after step 2, users would have no roles (old or new). In your proposal, users would have both old and new roles.

**Comment by andrestc** [comment]
I think that is probably safer

**Comment by ArxdSilva** [comment]
Did some improvements to what @andrestc suggested, I was ignoring the error on `user.AddRole()`, not anymore. Also moved `role.Name = newRoleName` to where It's really needed.

**Pull request merged**